projects/mtgo/index.js

```diff
@@ -0,0 +1,39 @@
+const { getBlock } = require("../helper/getBlock");
+const {
+    sumLPWithOnlyOneTokenOtherThanKnown,
+} = require("./../helper/unwrapLPs");
+
+const lpToken = "0x2a382b6d2dac1cba6e4820fd04e3c2c14e1aa7b2";
+const mtgoToken = "0x1bc8547e3716680117d7ba26dcf07f2ed9162cd0";
+const lpAddress = [ "0xdE2957506B6dC883963fbE9cE45a94a8A22c6006", "0x1781d2e9b4c7c0a3657411a64d2c1dfc50118772" ];
```

Comment:
`0x1781d2e9b4c7c0a3657411a64d2c1dfc50118772` looks like your staking address, can you remove the other one which looks like wallet, PR looks good otherwise

Reply:
0xdE2957506B6dC883963fbE9cE45a94a8A22c6006, This is the wallet address of the project party. The project party uses its own iotex to add the first batch of liquidity，You can see the transaction hash:dc9327be35b488c6f12bdf4580e0245dec0292f2051a280bef202c9be98af23c

Reply:
@yk200842135 yes, that is good but if you want you can add it as treasury not pool2, pool2 is what user staked LP

Reply:
Ok, I get it now, thanks! I will update